scripts/editor/register-blocks.js

```diff
@@ -314,63 +315,62 @@ export const prepareComponentAttribute = (component, realComponentName, newCompo
  * Iterate over component object in block manifest and check if the component exists in the project.
  * If components contains more component this function will run recursively.
  *
- * @param {object} componentsManifest              - Object of component manifests to iterate.
- * @param {object} blockManifest                   - Object of blocks manifests to iterate.
- * @param {string} blockName                       - Full block name.
- * @param {string} [key=attributes]                - Type of output, can be: `attributes` or `example`.
- * @param {string} [parentAttributeName=undefined] - Parent component attribute from which to determine if the name has changed in the parent component.
+ * @param {object} componentsManifest - Object of component manifests to iterate.
+ * @param {object} manifest           - Object of component/block manifest to get data from.
+ * @param {boolean} [isExample=false] - Type of items to iterate, if true example key will be use, if true attributes will be used.
```

Comment:
```suggestion
 * @param {boolean} [isExample=false] - Type of items to iterate, if true example key will be used, if false attributes will be used.
```

I'm assuming that the second case should be the false one in the statement